src/preprocess/index.ts

```diff
@@ -50,7 +50,7 @@ export function interpolateHandlebarsString(templateString: string, env: object,
   try {
     const template = handlebars.compile(templateString, {noEscape: true, strict: true});
     return template(env);
-  } catch (e) {
+  } catch (e: any) {
```

Comment:
would unkown be good choice for error

Reply:
Unknown is the default type here. I was considering doing some run-time type assertions rather than `any`. Going to look into that now before going out of `draft`.